runtime/compiler/x/amd64/runtime/AMD64CompressString_nasm.inc

```diff
@@ -0,0 +1,235 @@
+; Copyright (c) 2000, 2018 IBM Corp. and others
```

Comment:
These copyrights seem incorrect for the new files added.

Reply:
I have intentionally kept the first copyright dates according to how their masm-equivalent files had it. This is due to the fact that I've taken basically the same code and have re-written it in a different format in a new file. Eventually, the MASM files will be removed once Linux and Windows support is added.